src/plugin/coreplugins/RestRouterGenerator/Templates/combine_templates.js

```diff
@@ -0,0 +1,84 @@
+/*jshint node:true*/
```

Comment:
Yes you're right and I tried it too but ran into requirejs issues.
Looked closer at it and turns out if the name contains `.js` it throws the following error at the tests. 'Error: ENOENT: no such file or directory, open 'C:\GIT\webgme\plugin\RestRouterGenerator\RestRouterGenerator\RestRouter.js.ejs`... So I will just rename it.